src/connections/destinations/catalog/algolia-insights/index.md

```diff
@@ -24,31 +24,37 @@ This destination is maintained by [Algolia](https://www.algolia.com/). For any i
 2. Search for "Algolia" in the Catalog, select it, and choose which of your sources to connect the destination to.
 3. Enter the "App ID" & "API Key" into your Segment Settings UI which you can find on the Algolia Dashboard, under API Keys menu.
 
-_**NOTE:** The Algolia Insights Destination is not a plug-and-play integration. It requires you to modify your frontend code to send additional Algolia-related data like index name, queryID, etc._
+> _**NOTE:** The Algolia Insights Destination is not a plug-and-play integration. It requires you to modify your frontend code to send additional Algolia-related data like index name, queryID, etc._
 
+To access your queryID, make sure clickAnalytics are enabled in your search event. If you're using Insights.js this will look like
+```js
+index.search('query', {
+  userToken: 'user-1',
+  clickAnalytics: true
+})
+``` 
+
+Once this is enabled you will be able to send properties like queryId in your segment events. You can read more about how to send Algolia-related data to Segment from [the documentation at Algolia](https://www.algolia.com/doc/guides/sending-events/implementing/connectors/segment/).
+
+## Mapping Events
+
+By default, Algolia has set up mappings for Product Clicked, Product Viewed and Order Completed events. If your event structure doesn't match the spec you can update this by using the Mapping Tab.
```

Comment:
Read from top to bottom, the reader doesn't know what 'the spec' is at this point in the document. We should either more the explination of that up to this point from below.

> For a full list of required properties for each event type, see the [Spec: V2 Ecommerce Events](https://github.com/algolia/segment-docs/blob/18e5087751b4d5cf21f777c4bdb36de27e939288/docs/connections/spec/ecommerce/v2)

Or this should be reworded to not include a reference to 'the spec'.

It'd also be reasonable to simply repeat ourselves here and lower in the document where we use the fully worded "Spec: V2 Ecommerce Events" and link to the ecommerce 2.0 events spec.